Plugins/Optimizations/Optimizations/GameObjectLookup.cpp

```diff
@@ -304,4 +311,15 @@ uint8_t GameObjectLookup::GetGameObject(void*, uint32_t id, CGameObject** ptr)
     return BadId;
 }
 
+void GameObjectLookup::SyncWithGameGOA()
+{
+    if (auto pGameObjectArray = Globals::AppManager()->m_pServerExoApp->GetObjectArray())
```

Comment:
Nitpick: missing the `*` :P

Reply:
I copied it from your code. Fixed in both places :)